packages/storybook/stories/money-input.mdx

```diff
@@ -0,0 +1,38 @@
+import { MoneyInput } from '@equisoft/design-elements-react';
+import { ArgTypes, Canvas, Meta } from '@storybook/blocks';
+import * as MoneyInputStories from './money-input.stories';
+
+<Meta of={MoneyInputStories} />
+
+# Money Input
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Money input is a control element that allows the user to enter numeric values that represent an amount of money in a particular currency.
```

Comment:
On va simplifier:
"_Money input allows users to enter numeric values that represent an amount of money in a particular currency_"